presto-native-execution/presto_cpp/main/common/prometheus-metrics/PrometheusReporterTest.cpp

```diff
@@ -0,0 +1,154 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "PrometheusReporter.h"
+#include <gtest/gtest.h>
+
+namespace facebook::presto::prometheus {
+class PrometheusReporterTest : public testing::Test {
```

Comment:
This test will not run in the CI since that flag is off by default, can we do anything else to make it run?

Reply:
We may have to create a new CI job that builds code with prometheus enabled and run this test. Or, always build the code in existing CI jobs with prometheus enabled.

Reply:
Maybe just add a comment stating that this test is NoOp in CI for now